Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -843,7 +898,33 @@ static std::optional<CalendarRecord> parseCalendar(StringParsingBuffer<Character
     if (*buffer != ']')
         return std::nullopt;
     buffer.advance();
-    return CalendarRecord { WTFMove(result) };
+
+    return CalendarRecord { isCritical, false, WTFMove(result) };
+}
+
+template<typename CharacterType>
+static std::tuple<std::optional<CalendarRecord>, std::optional<CalendarRecord>>
+parseCalendar(StringParsingBuffer<CharacterType>& buffer)
+{
+    std::optional<CalendarRecord> first = parseOneCalendar(buffer);
```

Comment:
Ditto. Please add link and add BNF.

Reply:
Fixed in [19227db](https://github.com/WebKit/WebKit/pull/36361/commits/19227db637e5c5b34d4676a07f7a0147f9f991fa)